app/src/main/res/layout/test_bottom_sheet_options_menu.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <ImageButton
```

Comment:
Could this just be a Button or even a TextView? Does it need an image, or to be clickable?

Reply:
Removed File